### PR TITLE
MueLu/Ifpack2/ML: Improvements to Maxwell1

### PR DESCRIPTION
**Pull request by csiefer2**
Issue: #11214
Tests: Passes customer test
Stakeholder feedback: n/a

### CODE REVIEW

File: packages/muelu/src/Operators/MueLu_RefMaxwell_def.hpp [hunk 1]
```diff
@@ -364,31 +364,7 @@ namespace MueLu {
...
-        rapList.set("rap: fix zero diagonals", parameterList_.get<bool>("rap: fix zero diagonals", true));
-        rapList.set("rap: fix zero diagonals threshold", parameterList_.get<double>("rap: fix zero diagonals threshold", Teuchos::ScalarTraits<double>::eps()));
-        rapList.set("rap: triple product", parameterList_.get<bool>("rap: triple product", false));
-        rapFact->SetParameterList(rapList);
-
-
-        coarseLevel.Request("A", rapFact.get());
-
-        A_nodal_Matrix_ = coarseLevel.Get< RCP<Matrix> >("A", rapFact.get());
+        A_nodal_Matrix_ = Maxwell_Utils<SC,LO,GO,NO>::PtAPWrapper(OldSmootherMatrix,NodalP,parameterList_,labelstr);
```
Comment (cgcgcg) [inline packages/muelu/src/Operators/MueLu_RefMaxwell_def.hpp:367]: Is this being called with the correct arguments? Shouldn't that be `Ms_Matrix_`, `D0_Matrix_`? Where is `labelstr` defined?

File: packages/muelu/src/Operators/MueLu_Maxwell1_def.hpp [hunk 2]
```diff
@@ -107,10 +106,20 @@ namespace MueLu {
       // interpret ML list
       newList.sublist("maxwell1: 22list") = *Teuchos::getParametersFromXmlString(MueLu::ML2MueLuParameterTranslator::translate(list,"Maxwell"));
 
+
+      // Hardwiring options to ensure ML compatibility
+      newList.sublist("maxwell1: 22list").set("use kokkos refactor", false);
```
Comment (cgcgcg) [inline packages/muelu/src/Operators/MueLu_Maxwell1_def.hpp:111]: Would it make more sense to set that as default instead of hardwiring it?

Comment (cgcgcg) [reply packages/muelu/src/Operators/MueLu_Maxwell1_def.hpp:111]: Oh wait, this is the ML translation only.

File: packages/muelu/src/Operators/MueLu_RefMaxwell_def.hpp (hunk 1 above)
Comment (csiefer2) [reply packages/muelu/src/Operators/MueLu_RefMaxwell_def.hpp]: Copy pasta error!  Pushed fix

File: packages/muelu/src/Operators/MueLu_Maxwell1_def.hpp (hunk 2 above)
Comment (csiefer2) [reply packages/muelu/src/Operators/MueLu_Maxwell1_def.hpp:111]: Right now it is hardwired to use an ML style list too...

**Review by cgcgcg — APPROVED** [review]

**Comment by csiefer2** [comment]
> Looks like you want to perform a squash merge in the end ;-)

Yeah, probably.

**Pull request merged**